apidoc/Titanium/App/App.yml

```diff
@@ -293,8 +295,9 @@ events:
     summary: Fired when the application returns to the foreground.
     description: |
         This event fires when the application enters the foreground with the `resume` event or 
-        returns to focus after a notification or incoming phone call.
-        
+        returns to focus after a notification or incoming phone call. If building an iPad application
```

Comment:
"On iPad with split view enabled, this event will be called every time the application resized."